packages/cli/src/commands/doctor/healthchecks/common.ts

```diff
@@ -23,20 +38,58 @@ const logManualInstallation = ({
   }
 
   if (url) {
-    return logMessage(
+    logMessage(
       `Read more about how to download ${healthcheck} at ${chalk.dim.underline(
         url,
       )}`,
     );
+
+    return;
   }
 
   if (command) {
-    return logMessage(
+    logMessage(
       `Please install ${healthcheck} by running ${chalk.bold(command)}`,
     );
   }
 };
 
+const logError = ({
+  healthcheck,
+  loader,
+  error,
+  message,
+  command,
+}: {
+  healthcheck: string;
+  loader?: Ora;
+  error: Error;
+  message?: string;
+  command: string;
+}) => {
+  if (loader) {
+    loader.fail();
+  }
+
+  addBlankLine();
+
+  logMessage(chalk.dim(error.message));
+
+  if (message) {
+    logMessage(message);
+    addBlankLine();
+
+    return;
+  }
+
+  logMessage(
+    `The error above occured while trying to install ${healthcheck}. Please try again manually: ${chalk.bold(
+      command,
+    )}`,
+  );
+  addBlankLine();
```

Comment:
I think this is the one that causes extra space which is not necessary imho

Reply:
Which space do you mean? the one before `iOS` or the one after `iOS` is done?

Reply:
![image](https://user-images.githubusercontent.com/5106466/65597011-71df6680-df98-11e9-929f-bfc1d218f84c.png)

Reply:
That's because there's a space after each health check category (in this case `Common` and `iOS`) and also a space after each error is shown, that's why that extra space is there.

To be sincere I actually tried to get rid of it but there isn't an easy way to do at the moment, if we remove this last space then it can get to a funky UI when the previous health check succeeded but the next one failed.

Reply:
The best way to do this is to actually check if the last health check failed, if so then print only one space instead of two.

Reply:
Let's figure this out later, this is a very minor inconvenience

Reply:
Thanks for commenting about this though!